plugins/mission/mission_item_impl.cpp

```diff
@@ -85,9 +90,13 @@ float MissionItemImpl::get_mavlink_param1() const
 float MissionItemImpl::get_mavlink_param2() const
 {
     float acceptance_radius_m;
-    if (_fly_through) {
+    if (_acceptance_radius_m) {
```

Comment:
I don't like to check a float for `== 0`, so I'd rather check for `std::isfinite()` and then use it if it's set.

Reply:
got it